spec/GameSpec.js

```diff
@@ -7,25 +7,43 @@ describe('Game', function() {
     expect(game.overallScore).toEqual(0);
   });
 
-  it('generates all the frames', function() {
+  it('generates all the frames when there are no strikes or spares at the end', function() {
     expect(game.loadFrames([3, 5, 10, 5, 5, 3, 5, 6, 2, 2, 3, 5, 4, 7, 1, 7, 1, 6, 2]).length).toEqual(10);
   });
 
```

Comment:
Consider putting these in a describe block with a before element before the 3 tests which could represent the first 9 frames of each scenario

Reply:
To clarify, you could create a function:  

	var rollX = function(pins, rolls) {
		for (var i = 0; i < rolls; i++)
			game.roll(pins);
	};
Then before each scenario do rollX(1,18) to roll 18 individual 1's, leaving your final frame clean for manipulation with a strike or spare